modules/transfer-frontend/templates/cognito_authenticated.json.tpl

```diff
@@ -0,0 +1,42 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "ListYourObjects",
+      "Effect": "Allow",
+      "Action": "s3:ListBucket",
+      "Resource": [
+        "arn:aws:s3:::tdr-upload-files-${environment}"
+      ],
+      "Condition": {
+        "StringLike": {
+          "s3:prefix": [
+            "$${cognito-identity.amazonaws.com:sub}"
```

Comment:
Should that be a "$$"?

Reply:
Yeah, because the string in the policy needs to be `${cognito-identity.amazonaws.com:sub}` but terraforms string interpolation uses ${} This is how you get it to just use the string you want in the policy.

Reply:
Fair enough